pkgs/games/dopewars/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, gtk3, autoconf, automake, ncurses, curl }:
+
+stdenv.mkDerivation {
+  pname = "dopewars";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "benmwebb";
+    repo = "dopewars";
+    rev = "86c3e8a841454440966693d571899fe4f738a4ae";
+    sha256 = "sha256-CpgqRYmrfOFxhC7yAS2OqRBi4r3Vesq3+7a0q5rc3vM=";
+  };
+
+  nativeBuildInputs = [ pkg-config autoconf automake curl ];
+  buildInputs = [ gtk3 ncurses ];
+
+  patchPhase = ''
+  sed -i 's/chmod 2755/chmod 755/' ./src/Makefile.am
```

Comment:
Use a proper patch file here rather than a sed command. When Makefile.am changes, the patch file will cause the build to fail with an error message, whereas sed will silently fail.